_episodes/02-filedir.md

```diff
@@ -273,7 +273,7 @@ Mandatory arguments to long options are mandatory for short options too.
                                file-type (--file-type), classify (-F)
   -i, --inode                print the index number of each file
   -I, --ignore=PATTERN       do not list implied entries matching shell PATTERN
-  -k, --kibibytes            default to 1024-byte blocks for disk usage
+  -k, --kilobytes            default to 1024-byte blocks for disk usage
```

Comment:
Hi, this isn't a typo. The manpage says this, and this is a different unit: https://en.wikipedia.org/wiki/Kibibyte

Can you please update the PR to remove this.